client/my-sites/customer-home/cards/actions/wp-for-teams-quick-links-compact/index.jsx

```diff
@@ -0,0 +1,226 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
This one is also mostly a copy-paste from [its wide counterpart version](https://github.com/Automattic/wp-calypso/blob/5fb0b042c067a32feaa9a4f00d8b7e90969cc95b/client/my-sites/customer-home/cards/actions/wp-for-teams-quick-links/index.jsx) just to ensure we use `FoldableCard` component.